### ISSUE TITLE
Release on September 23, 2016

### ISSUE THREAD

**Issue by nzakas** [eslint/eslint#7038]
The scheduled release on Friday, September 23, 2016 is assigned to:

* @gyandeeps 
* @ilyavolodin

This issue should be closed when the release has been completed.

**Comment by ilyavolodin** [comment]
Release went smoothly overall. After minor release on Friday, we found an issue with Espree and did a follow-up patch release on Monday to upgrade Espree version. There are still 2 outstanding regressions from the Friday build that were reported, but still do not have a fix available. Right now, the main problem is not knowing if we should do another follow-up patch release after those issues are fixed, or if we should wait for normal release in 2 weeks.

**Comment by gyandeeps** [comment]
I think the release very smooth and easy. Thanks for automating all this @nzakas .

**Comment by nzakas** [comment]
While the initial release seems to have gone smoothly, I think the aftermath has been a bit messy. What I've picked up from the chat:

1. People were unsure if there was going to be a patch release on Monday. 
2. There was a lot of confusion over whether or not there should be a second patch release. That confusion continued into Wednesday, at which point at least one semver-minor PR had been merged, which meant we couldn't do a second patch release and instead would have to either do another non-patch release or wait until the next regularly scheduled release.

I think we should do a post-mortem on this release during the TSC meeting to find out what happened and what we can do better to manage the Monday/Tuesday after regular releases.
